app/main/index.js

```diff
@@ -151,6 +151,54 @@ app.on('ready', () => {
 
 	const page = mainWindow.webContents;
 
+	const ses = page.session;
+
```

Comment:
I think we could put this into a `proxy-util.js`.

Reply:
Do you mean the entire thing from here till the [end](https://github.com/zulip/zulip-electron/pull/480/files#diff-b7ea71851f2bf6f64dbcec17f174a921R201)? 
I will try to do that. We can use it as `proxyUtil.setSystemProxy()` ?

Reply:
I  think `storeSystemProxy`/`resolveSystemProxy` sounds appropirate.